src/containers/shared/components/Amount.tsx

```diff
@@ -22,7 +27,10 @@ export const Amount = ({ displayIssuer = true, value }: AmountProps) => {
 
   return (
     <span className="amount">
-      <span className="amount-localized">{localizedAmount}</span>{' '}
+      <span className="amount-localized">
+        {modifier && <span className="amount-modifier">{modifier}</span>}
```

Comment:
What kind of modifiers are there? just `+` and `-`?

Reply:
So far I have only used `+`, but it could be `~` or `-`.  I suppose i could limit it to those 3.